Support qiskit version 0.25 (#132)
PennyLaneAI/pennylane-qiskit: doc/devices/aer.rst

Backends
~~~~~~~~

> The default backend is the ``AerSimulator``. However, multiple other backends might be available.

Reviewer: (minor)
```suggestion
The default backend is the ``AerSimulator``. However, multiple other backends are also available.
```

.. code-block:: python

    dev = qml.device('qiskit.aer', wires=2, backend='aer_simulator_statevector')

Backend Methods and Options
~~~~~~~~~~~~~~~~~~~~~~~~~~~

This ``AerSimulator`` backend has several available methods, which
can be passed via the ``method`` keyword argument. For example

> ``'automatica'``, ``'statevector'``, and ``'unitary'``.

Reviewer 2: ```
This plugin supports the ``'automatic'``, ``'statevector'``, ``'density_matrix'``, and ``'unitary'`` methods.
```

I tested the methods in qiskit documentation.  ``"stabilizer"``, ``"extended_stabilizer"``, ``"matrix_product_state"``, and ``"superop"`` all raise errors and fail.

Author: This is a very good catch, I'll update the documentation and validation.

.. code-block:: python

    dev = qml.device('qiskit.aer', wires=2, backend='aer_simulator_statevector')

Backend Methods and Options
~~~~~~~~~~~~~~~~~~~~~~~~~~~

This ``AerSimulator`` backend has several available methods, which
can be passed via the ``method`` keyword argument. For example
``'automatic'``, ``'statevector'``, and ``'unitary'``.

.. code-block:: python

    dev = qml.device("qiskit.aer", wires=2, method="automatic")

Each of these methods can take different *run options*, for example to specify the numerical
precision of the simulation.

The options are set via additional keyword arguments:

.. code-block:: python

    dev = qml.device(
        'qiskit.aer',
        wires=2,
        backend='unitary_simulator',
        validation_threshold=1e-6
    )

For more information on available methods and their options, please visit the `AerSimulator
documentation <https://qiskit.org/documentation/stubs/qiskit.providers.aer.AerSimulator.html>`_.

> .. warnings::

Reviewer: ```suggestion
.. warning::
```
